modules/op-tkn-pipeline-management.adoc

```diff
@@ -14,15 +14,6 @@ Manage Pipelines.
 $ tkn pipeline --help
```

Comment:
All three download links in this doc need to be updated to https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.11.0/